build.gradle

```diff
@@ -0,0 +1,74 @@
+plugins {
+    id 'maven-publish'
+    id 'net.researchgate.release' version '2.3.4'
+    id "com.jfrog.bintray" version "1.7"
+    id 'java-gradle-plugin'
+}
+
+repositories {
+    jcenter()
+}
+
+def actualVersion = "$version" // remapping version to this var, since version has namespace clashes in some cases (e.g. within bintray.pkg.version)
+def scmTag='v'+actualVersion
```

Comment:
This is only used in the `bintray` section and should be declared there. Also `"v$project.version"` reads more nicely.